### PR TITLE
enable XEH for classes used on map (but not terrain objects)

### PR DESCRIPTION
**Pull request by commy2**
**When merged this pull request will:**
- enable XEH / `CBA_fnc_addClassEventHandler` for many additional classes, mainly stuff like modules, game logics, furniture, pop up targets...
- terrain objects (that belong to the map) are still disabled. this is done already by the scripting, because `typeOf` reports `""` apparently for them.
- this does not enable for `class Static` objects (houses, bushes, trees, rocks, road signs, flagpoles, hesco barriers ...)

Will probably make a ton of XEH not supported for X appear with other mods (e.g. ACE), because, well they don't, and were peviously disabled. Nothing that can't be fixed by the authors though.

### CODE REVIEW

File: addons/xeh/CfgVehicles.hpp [hunk 1]
```diff
@@ -137,7 +213,7 @@ class CfgVehicles {
         XEH_ENABLED;
     };
 
-    // backwards comp, inert
+    // backwards comp, inertx
```
Comment (Killswitch00) [inline addons/xeh/CfgVehicles.hpp]: "inertx" ?

Comment (commy2) [inline addons/xeh/CfgVehicles.hpp]: lol
I think this was a failed attempt at ctrl + x.

**Comment by Killswitch00** [comment]
This will need thorough testing with both current and older missions and addons.

**Comment by commy2** [comment]
Thanks to #663, terrain objects are now explicitly skipped.

**Comment by commy2** [comment]
Merge for RC? Can always undo _if_ it causes problems.

**Comment by Killswitch00** [comment]
Yes, let's merge for the RC.

**Pull request merged**